index.html

```diff
@@ -0,0 +1,234 @@
+        <div class="about-me">
+            <h2>About me</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button>Get My Resume</button>
+          </div>
+          <hr>
+          <div class="skills-container">
+
+            <div class="skills-title">
+              <svg width="27" height="26" viewBox="0 0 27 26" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect x="0.772461" y="12.7279" width="18" height="18" transform="rotate(-45 0.772461 12.7279)" fill="#FF6B00"/>
+              </svg>
+              <h2>Language</h2>
+            </div>
+            <ul>
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Css</li>
+            </ul>
+            
+            <div class="skills-title">
+              <svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect y="0.455811" width="1f" height="18" fill="#FF6B00"/>
+              </svg>    
+              <h2>Frameworks</h2>
+            </div>
+            
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+            
+            <div class="skills-title">
+              <svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect y="0.455811" width="18" height="18" rx="9" fill="#FF6B00"/>
+              </svg>     
+              <h2>Skills</h2>
+            </div>
+            <ul>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>         
+ 
+
+    </section>
+    <!-- contact-form section. -->
+    <section class="contact-section">
+            <p class="contact-head">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            <form class="form" action="https://formspree.io/f/mvonaqyb" method="POST">
+              <div>
+                <input class="form-input" type="text" name="name" placeholder="First name" maxlength="30" required>
+                <input class="form-input" type="text" name="name" placeholder="Last name" maxlength="30" required>
+              </div>
+                <input class="form-input" type="email" name="email" placeholder="Email address" maxlength="30" required>
+                <textarea name="mail-message" id="" cols="20" rows="5" placeholder="Enter text here" maxlength="500" required></textarea>
+                <input class="send" type="submit" value="Get in touch">
+            </form>
+    </section>
+    <footer>
```

Comment:
- [ ] Great job on implementing the form! :+1: However, ensuring that it closely matches the Figma template as much as possible is essential. Right now, I can't see the styles for your form. Remember that:
- Flexbox should be used to layout the elements.
- Students should stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding).
- Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) should be styled as shown in the Figma template.

**Your design**

![Screenshot from 2023-03-01 07-33-46](https://user-images.githubusercontent.com/105117832/222141020-25c08fe2-b3af-4f17-a43c-cdf51d3dcbcf.png)

**Figma**

![Screenshot from 2023-03-01 07-35-06](https://user-images.githubusercontent.com/105117832/222141065-590155ee-4826-4ac5-b2c8-9dc433c91adc.png)